proto/zitadel/admin.proto

```diff
@@ -6563,48 +6563,48 @@ message SetDefaultVerifyPhoneMessageTextRequest {
         }
     ];
     string title = 2 [
-        (validate.rules).string = {max_len: 200},
+        (validate.rules).string = {max_bytes: 2000},
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "\"ZITADEL - Verify Phone\""
-            max_length: 200;
+            max_length: 500;
         }
     ];
     string pre_header = 3 [
-        (validate.rules).string = {max_len: 200},
+        (validate.rules).string = {max_bytes: 2000},
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "\"Verify Phone\""
-            max_length: 200;
+            max_length: 500;
         }
     ];
     string subject = 4 [
-        (validate.rules).string = {max_len: 200},
+        (validate.rules).string = {max_bytes: 2000},
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "\"Verify Phone\""
-            max_length: 200;
+            max_length: 500;
         }
     ];
     string greeting = 5 [
-        (validate.rules).string = {max_len: 200},
+        (validate.rules).string = {max_bytes: 4000},
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "\"Hello {{.FirstName}} {{.LastName}},\""
-            max_length: 200;
+            max_length: 1000;
         }
     ];
     string text = 6 [
-        (validate.rules).string = {max_len: 800},
+        (validate.rules).string = {max_bytes: 40000},
```

Comment:
as this is sent through sms, we should let its stay as is (and maybe drop the unused fields)